keps/sig-node/3675-resource-plugin-manager/README.md

```diff
@@ -0,0 +1,1002 @@
+
+Yes, a deletion of pods requiring cci driver will be recommended
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+Running pods should not be impacted.  New pods should be handled by the CCI Driver.
+
+###### Are there any tests for feature enablement/disablement?
+
+Yes.
+
+### Rollout, Upgrade and Rollback Planning
+
+The usual Kubernetes upgrade and downgrade strategy applies for in-tree components. 
+Vendors must take care that upgrades and downgrades work with the drivers that
+they provide to customers
+  
+Pods which require CCI Driver will fail starting and report an error due to driver 
+unavailability.  If a CCI Driver fails, Pods which were about to be allocated
+that were tied to the driver will fail. Based on requirements a fallback mechanism 
+can be implemented where such Pods fallback to a given std. QoS Model.
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+Failure of CCI drivers will have impact only over pods requiring CCI driver for 
+resource management. All other pods will not be impacted. THe impacted pods will
+an error message showing failure of the driver.
+
+
+###### What specific metrics should inform a rollback?
+
+Unhealthy CCI drivers which can't get to a ready state and repeated failures 
+to allocate pods in cases of free resources.
+
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+TBD in Beta
+
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+TBD in Beta
+
+
+### Monitoring Requirements
+
+TBD in Beta
+
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Pods will be running on the cluster which are associated to CCI drivers. 
+Additionally, available resources will be updated.
+
+
+###### How can someone using this feature know that it is working for their instance?
+
+- [ ] Events
+  - Event Reason: 
+- [x] API .status
+  - Condition name: CCI Driver readiness and CCI Status field can be introduced
+  - Other field: 
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+Pods not using CCI Drivers continue to work as before. Consistent state handling for 
+pods which require a CCI Driver.
+
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+<!--
+Pick one more of these and delete the rest.
+-->
+TBD in Beta
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+<!--
+Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+implementation difficulties, etc.).
+-->
+TBD in Beta
+
```

Comment:
I think we should populate this section and aim to introduce metrics that are indicative of the health of the feature.

Reply:
done , i added analog to cpu manager the following counters : cci_manager_allocation_requests_total, cci_manager_allocation_errors_total